config/initializers/active_model_serializer.rb

```diff
@@ -0,0 +1,4 @@
+# Globally disable the root element when rendering json
+# Beware, this 'root = false' doesn't seem to have an effect.
+# JE Todo: Understand why.
+ActiveModel::Serializer.root = false
```

Comment:
That still didn't seem to influence the controller rendered json.  I've just gone ahead and removed this file, since the default_serializer_options in the ApplicationController takes care of things.